### PR TITLE
New: Default severity for `hint-webpack-config`

### LINKED ISSUE #3253: [Severity Updates] hint-webpack-config
| Report | Severity |
| --------| ----------|
| Report message when modules option si not false: Babel presets `modules` option should be `false` | Error |
| Report message when a webpack configuration file doesn't exists: webpack configuration file not found in your project. | Off | 
| Report message when the compilerOptions.module doesn't have esnext configured: TypeScript `compilerOptions.module` option should be `esnext` | Error |
| Report message when webpack is not installed in your project: webpack is not installed in your project. | Warning | 
| Report message when devtool = eval is configured for production: `$1` not recommended for prodution | Warning |

### PR DESCRIPTION
**Pull request by sarvaje**
Fix #3253

## Pull request checklist

Make sure you:

- [x] Signed the [Contributor License Agreement](https://cla.js.foundation/webhintio/hint)
- [x] Followed the [commit message guidelines](https://webhint.io/docs/contributor-guide/getting-started/pull-requests/#commit-messages)

For non-trivial changes, please make sure you also:

- [ ] Added/Updated related documentation.
- [x] Added/Updated related tests.

### CODE REVIEW

File: packages/hint-webpack-config/src/config-exists.ts [hunk 1]
```diff
@@ -25,7 +25,11 @@ export default class WebpackConfigConfigExists implements IHint {
         const notFound = () => {
             debug(`'parse::error::webpack-config::not-found' received`);
 
-            context.report('', getMessage('configExists', context.language));
+            context.report(
```
Comment (molant) [inline packages/hint-webpack-config/src/config-exists.ts]: I've been treating `off` as deletions of tests unless the severity can change for another factor. I think we want to delete this tests and just make the other ones conditional to receive the configuration.

Comment (sarvaje) [reply packages/hint-webpack-config/src/config-exists.ts]: this hint only check for that, do we want to delete the whole hint?

**Comment by molant** [comment]
Or we delete it or we have it off by default in the configurations and report as an error if enabled.

I'm OK either way. What do you think @antross?


________________________________

File: packages/hint-webpack-config/src/config-exists.ts (hunk 1 above)
Comment (antross) [reply packages/hint-webpack-config/src/config-exists.ts]: Honestly, I'm okay with deleting the whole hint. I don't think this one is adding value.

Comment (sarvaje) [reply packages/hint-webpack-config/src/config-exists.ts]: ok, I'm going to delete it

**Comment by sarvaje** [comment]
@molant @antross This is ready to review

**Pull request merged**